### ISSUE TITLE
Call HtmlHelp instead of WinHelp in the Help menu

### ISSUE THREAD

**Issue by daviunic** [Microsoft/winfile#60]
HtmlHelp (.chm files) still works in all versions of Windows, unlike WinHelp (.hlp files). Switching to it is rather easy, but it requires include of htmlhelp.h and linking against htmlhelp.lib (both come with HTML Help Workshop 1.3). Not sure how acceptable that is in this case, though.

**Comment by Anixx** [comment]
The .hlp file works well for me on Windows 8.1.
To make the .hlp files work one has to install some missing Windows components (an optional update) from Microsoft. That's why I think the .hlp file should be included in the release.
![clipboard_20180410](https://user-images.githubusercontent.com/1271736/38570009-4ad0abf4-3cdc-11e8-9950-a776fbec2b8e.png)

**Comment by daviunic** [comment]
This isn't available for Windows 10 and can't be installed on it. This leaves you with manually obtaining winhlp32.exe from a previous Windows version, which is far from ideal.

**Comment by ZanderBrown** [comment]
Is there a way we can use the old `.hlp` by default and fallback to `.chm`?

**Comment by daviunic** [comment]
Yeah, if the call to WinHelp fails you could make a call to HtmlHelp and if even that fails, display the error message you would usually get. But as said, for calling HtmlHelp you have to link against htmlhelp.lib.

**Comment by yuhong** [comment]
I wonder how Windows 98 called HTML Help from 16-bit WinFile.

**Comment by ZanderBrown** [comment]
Carefully? Believe or not you can mix 16/32bit if you know what your doing

**Comment by craigwi** [comment]
I'm not sure the direction folks want to take here.  As mentioned in various places, the hlp files only work on Windows <= 8.1.  Perhaps the ver9x branch can just use the code as is and the master branch (>= Vista?) would need someone to propose a specific alternative.
